app/controllers/comments_controller.rb

```diff
@@ -15,6 +15,12 @@ def create
     end
   end
 
+  def destroy
+    @comment = Comment.find(params[:comment_id])
+    @comment.destroy
+    redirect_to user_post_path(@comment.author), notice: 'Comment was deleted succesfully'
+  end
+
```

Comment:
- [ ] The same happens on this controller, please consider loading and authorizing the resource! 🤓 🔧

Reply:
added for both, thanks for the observation!